src/com/gmail/chernii/oleksii/flowers/Flower.java

```diff
@@ -0,0 +1,7 @@
+package com.gmail.chernii.oleksii.flowers;
+
+/**
+ * Created by Space on 11.03.2019.
+ */
+public class Flower {
```

Comment:
it should be abstract. And add field price here, getter and constructor with a parameter.